docs/tutorials/python-development-environment.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Setting up a Python development environment
+subject: Software
+---
+By Anton Vanhoucke
```

Comment:
`author: Your Name` in the frontmatter would be good. 

I kind of like idea of having the author displayed on a tutorial though (other than the very basic getting started related tutorials). You could even do a badge like on the project pages.